src/controller.js

```diff
@@ -0,0 +1,18 @@
+class Controller{
+    constructor(){
+        this.initializeSea();
+    }
+    initializeSea() {
+        let counter = 0;
+        setInterval( () => {
+       const viewport = document.querySelector("#viewport");       
+       if(counter % 2 === 0){
+        viewport.style.backgroundImage = "url('../images/water1.png')"; 
+       counter ++;
```

Comment:
`counter++;` could have been outside the if/else block as it needs to be applied no matter what...

```
setInterval(() => {
  const viewport = document.querySelector("#viewport");
  if (counter % 2 === 0) {
    viewport.style.backgroundImage = "url('../images/water1.png')";
  } else {
    viewport.style.backgroundImage = "url('../images/water0.png')";
  }
  counter++;
}, 1000);
```